src/client.ts

```diff
@@ -372,6 +372,15 @@ export default class Client {
     target: Target,
     defaultValue = '',
   ): Promise<string> {
+    if (!this.initialized) {
+      warnDefaultVariationServed(
+        identifier,
+        target,
+        defaultValue.toString(),
+        this.log,
+      );
+      Promise.resolve(defaultValue);
```

Comment:
No point trying to evaluate if we haven't finished initializing, so just return default value immediately.

Reply:
I have ran further manual tests with `waitForinit` called and not called.